MapleServer2/Packets/SkillDamagePacket.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Maple2Storage.Types;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Types;
+
+namespace MapleServer2.Packets
+{
+    public static class SkillDamagePacket
+    {
+        public static Packet SkillDamage(IFieldObject<Player> player, long skillCount, int someValue, int skillId, short skillLevel, CoordF coords, int objectId)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.SKILL_DAMAGE);
+            pWriter.WriteByte(1);
+            pWriter.WriteLong(skillCount);
+            pWriter.WriteInt(someValue);
+            pWriter.WriteInt(player.ObjectId);
+            pWriter.WriteInt(player.ObjectId);
+            pWriter.WriteInt(skillId);
+            pWriter.WriteShort(skillLevel);
+            pWriter.WriteShort(1);
+            pWriter.Write(coords);
+            pWriter.WriteInt();
+            pWriter.WriteByte(9); // Heal or Damage
+            pWriter.WriteByte(1); //
+            pWriter.WriteByte(1); // How many mobs where hit
```

Comment:
Should this be an parameter or it doesn't matter?

Reply:
the value of that should be use as a parameter because is getting a count foreach mob was hit at the moment, but for now is just being test for 1 mob

Reply:
Good, just making sure!